mimic/rest/maas_api.py

```diff
@@ -30,6 +30,16 @@
 Request.defaultContentType = 'application/json'
 
 
+def _find_index_of_id(xs, val):
```

Comment:
Tested, and actually it's `ValueError` - I'm going to fix my code to return that.